src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5644,4 +5644,10 @@
   <data name="WRN_CallerArgumentExpressionAttributeHasInvalidParameterName_Title" xml:space="preserve">
     <value>The CallerArgumentExpressionAttribute applied to parameter will have no effect. It is applied with an invalid parameter name.</value>
   </data>
+  <data name="ERR_BadAbstractStaticMemberAccess" xml:space="preserve">
+    <value>A static abstract interface member cannot be accessed.</value>
```

Comment:
>static abstract [](http://example.com/codeflow?start=13&length=15)

Should we use Shared MustInherit here? And same for the abstract in the message below?

Reply:
>Should we use Shared MustInherit here? And same for the abstract in the message below?

I don't think using ```MustInherit``` term is appropriate here because the language doesn't have such concept for interface members. I could replace ```static``` with ```shared``` for consistency though. Would that address the concern?

Reply:
Alright, as long as we're consistent between the error messages I'm ok.